src/php/bin/generate_proto_php.sh

```diff
@@ -28,13 +28,32 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-
 set +e
-cd $(dirname $0)
+cd $(dirname $0)/../../..
+
+protoc --proto_path=src/proto/math \
+       --php_out=src/php/tests/generated_code \
+       --grpc_out=src/php/tests/generated_code \
+       --plugin=protoc-gen-grpc=$(which grpc_php_plugin) \
```

Comment:
Actually, to regenerate code in our repository (not in examples), you should use the plugins from 
bins/opt/grpc_php_plugins (they'll be there after make plugins) - otherwise you'll be generating code with a different version of protobuf than the one in third_party/protobuf (and that's the one that should be used).